docs/cosmology/mypackage/io/astropy_convert.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see Astropy LICENSE.rst
+
+"""
+Register conversion methods for cosmology objects with Astropy Cosmology.
+
+With this registered, we can start with a Cosmology from
+``mypackage`` and convert it to an astropy Cosmology instance.
+
+    >>> from mypackage.cosmology import myplanck
+    >>> from astropy.cosmology import Cosmology
+    >>> cosmo = Cosmology.from_format(myplanck, format="mypackage")
+    >>> cosmo
+
+We can also do the reverse: start with an astropy Cosmology and convert it
+to a ``mypackage`` object.
+
+    >>> from astropy.cosmology import Planck18
+    >>> myplanck = Planck18.to_format("mypackage")
+    >>> myplanck
+
+"""
+
+# THIRD PARTY
+import astropy.cosmology.units as cu
+import astropy.units as u
+from astropy.cosmology import FLRW, Cosmology, FlatLambdaCDM
+from astropy.io import registry as io_registry
+
+# LOCAL
+from mypackage.cosmology import MyCosmology
+
+__doctest_skip__ = ['*']
+
+
+def from_mypackage(mycosmo):
+    """Load `~astropy.cosmology.Cosmology` from ``mypackage`` object.
+
+    Parameters
+    ----------
+    mycosmo : `~mypackage.cosmology.MyCosmology`
+
+    Returns
+    -------
+    `~astropy.cosmology.Cosmology`
+    """
+    m = dict(mycosmo)
+    m["name"] = mycosmo.name
+
+    # ----------------
+    # remap Parameters
+    m["H0"] = m.pop("hubble_parameter") * (u.km / u.s / u.Mpc)
+    m["Om0"] = m.pop("initial_matter_density")
+    m["Tcmb0"] = m.pop("initial_temperature") * u.K
+    m["Neff"] = m.pop("Neff")
```

Comment:
Like below, perhaps not change things that already are OK?

Reply:
SGTM.
Done!